src/schema/notifications.ts

```diff
@@ -219,13 +219,17 @@ export class ZimbraNotifications {
 
 	private processContactNotifications = (items: any) => {
 		let searchResponse: any = {};
+		// If tag or tagName present in notification item, skip notification updation
```

Comment:
We're optimistically updating cache on client side.

Reply:
optimistic update should be ok but we shouldn't ignore updates coming in notifications
consider scenario, user has updated from outlook and expecting that change to be reflected in webapp without reload

Reply:
Ah.. Not considered that case. Thanks. 👍